src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -685,7 +682,7 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <value>Refresh</value>
   </data>
   <data name="ShowError_ApplySettingFailed" xml:space="preserve">
-    <value>Applying setting failed.</value>
+    <value>Applying setting failed. Please report this issue through VSFeedback.</value>
```

Comment:
Failed to apply NuGet Package Manager settings. Please report a problem using Help > Send Feedback > Report a Problem.